core/templates/pages/preferences-page/modal-templates/edit-profile-picture-modal.controller.spec.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for EditProfilePictureModalController.
+ */
+
+import $ from 'jquery';
+import Cropper from 'cropperjs';
+
+describe('EditProfilePictureModalController', function() {
+  var $scope = null;
+  var $timeout = null;
+  var $uibModalInstance = null;
+
+  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.inject(function($injector, $controller) {
+    var $rootScope = $injector.get('$rootScope');
+    $timeout = $injector.get('$timeout');
+
+    $uibModalInstance = jasmine.createSpyObj(
+      '$uibModalInstance', ['close', 'dismiss']);
+
+    $scope = $rootScope.$new();
+    $controller('EditProfilePictureModalController', {
+      $scope: $scope,
+      $uibModalInstance: $uibModalInstance
+    });
+  }));
+
+  it('should init the variables', function() {
+    expect($scope.uploadedImage).toBe(null);
+    expect($scope.croppedImageDataUrl).toBe('');
+    expect($scope.invalidImageWarningIsShown).toBe(false);
+  });
+
+  it('should get a file in onchange event and save it', function(done) {
+    document.getElementById = jasmine.createSpy('HTMLImageElement').and
+      .returnValue(document.createElement('img'));
+
+    var element = $(document.createElement('div'));
+
+    // This is just a mock base 64 in order to test the FileReader event.
+    var dataBase64Mock = 'VEhJUyBJUyBUSEUgQU5TV0VSCg==';
+    const arrayBuffer = Uint8Array.from(
+      window.atob(dataBase64Mock), c => c.charCodeAt(0));
+    var file = new File([arrayBuffer], 'filename.mp3');
+
+    // @ts-ignore
```

Comment:
Explain the usage of `// @ts-ignore` please.

Reply:
@marianazangrossi ping.

Reply:
Done.